internal/utils/config.go

```diff
@@ -347,6 +349,17 @@ type (
 		TestOTP             map[string]string `toml:"test_otp"`
 	}
 
+	hook struct {
+		MFAVerificationAttempt      extensibilityPointConfig `toml:"mfa_verification_attempt" mapstructure:"mfa_verification_attempt"`
+		PasswordVerificationAttempt extensibilityPointConfig `toml:"password_verification_attempt" mapstructure:"password_verification_attempt"`
+		CustomAccessToken           extensibilityPointConfig `toml:"custom_access_token" mapstructure:"custom_access_token"`
+	}
+
+	extensibilityPointConfig struct {
```

Comment:
I'm tempted to name this struct `hookConfig` for simplicity. Wdyt?

Reply:
sounds good